ocagent.go

```diff
@@ -336,6 +336,14 @@ func (ae *Exporter) ExportSpan(sd *trace.SpanData) {
 	_ = ae.traceBundler.Add(sd, 1)
 }
 
+func (ae *Exporter) ExportSpanBatch(batch *agenttracepb.ExportTraceServiceRequest) error {
```

Comment:
How about making this method name `ExportTraceServiceRequest`?

But also am curious why this method at all? Are you encountering something during and just need to be able to use the retry logic?

Reply:
+1 to this.